added github action set up

Mikelobi opened the pull request:
Checking to see if github action configuration is ok

Mikelobi committed:
added github action set up

Amrhub commented on .gitignore:
```diff
@@ -0,0 +1 @@
+*.md
```
- [ ] I would suggest adding `node_modules` to our `.gitignore`, I know we are not using any dependency yet but that would be good practice for us since any future project we will work on will be using some what outside dependancy

Mikelobi replied:
I have updated it

Amrhub commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
- [ ] Let's also make sure we add `README.md` file that will make our project documented in a professional way. I would recommend using [Microverse README.md template](https://github.com/microverseinc/readme-template/blob/master/README.md)

Mikelobi replied:
It has been checked and fixed. Thank you

mateo951 replied:
Don't forget to add these sections 🧐 They give your readme essential information so that future interested users can have important social links and info.

![image](https://user-images.githubusercontent.com/15807343/153039737-a41f5336-5cec-49f6-ac7a-5884661169c1.png)

Mikelobi replied:
I have updated it

Amrhub reviewed (changes requested):
Hi @Mikelobi, Welcome to Microverse 👋🏻 ❤️ 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mikelobi committed:
added gitignore file

Mikelobi committed:
added html index file

Header of the file

Mikelobi committed:
added css file

changed the header colour

Mikelobi committed:
add class to the h1

Mikelobi committed:
updated the class name to h1

mateo951 commented on .gitignore:
```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules
```
Addressed ✔

mateo951 reviewed (approved):
### Hi @Mikelobi, 

**Having a complete README file will boost your project's exposure in the near feature so be sure to make them as clean and professional as possible 😉**

Nice work implementing all the changes requested by your previous reviewer ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mikelobi committed:
updated gitignore file

- added node_modules future project we will work on will be using some what outside dependancy.
- checked only the .md file I want

Mikelobi committed:
Added content to file

- This is information section that summarizes the project work

Mikelobi committed:
add about the author

Useful links to read more about the author of the project
